Add bio pages

HobnobMancer opened the pull request:
This is an enhancement or feature.

## Summary

Add first group member bio page

## Context

- Add first draft of bio page for Emma
- Establish bio page template
- Add first group member bio
- Link new bio page to the Group members page

HobnobMancer committed:
add bio page for Emma

first draft of bio page

HobnobMancer committed:
link group members page to Emma's bio

HobnobMancer committed:
update group info desc

'Resources for group' does not describe that these pages also contain information about the group, which could be useful information for prospective students/collaborators/employers to view

widdowquinn commented on _pages/bio_emmah.md:
You could link from here to the pages for those workshops, if you liked.

widdowquinn committed:
update list of group members

Two main changes

- add Julianna to the list (sorry, Julianna!)
- qdd GitHub avatars for group members who have accounts

widdowquinn committed:
make minor change to wording for group.md link

widdowquinn committed:
place group Bios under a new category

This change adds a new biography for Leighton, and adds the
`categories` field to the headers for Emma's and Leighton's
bios. This allows them to be picked up as a new collection at
the /bios/ permalink (bios.md). This required small changes
in group.md.

widdowquinn committed:
add categories page at /group/bios

widdowquinn commented:
Thanks Emma, that's great. I've extended things a bit by:

- adding my own Bio page
- adding the `categories` field to the pages, so we can automatically group them under `/group/bios` and `/bios/`
- added the category pages to support viewing them under `/group/bios` and `/bios/`

I also added GitHub avatars for people, where I could.
